test/ng/directive/ngModelOptionsSpec.js

```diff
@@ -459,11 +459,12 @@ describe('ngModelOptions', function() {
           $rootScope.$watch(watchSpy);
 
           helper.changeInputValueTo('a');
+          expect(function() {$browser.defer.flush();}).toThrowError('No deferred tasks to be flushed');
```

Comment:
The assumption that there are no deferred tasks is wrong at this point, because there's obviously a pending timeout.
We just need to check that the watchFn doesn't get called. 
If you put in this:
```
          helper.changeInputValueTo('a');
          $timeout.flush(2000);
          helper.changeInputValueTo('b');
          $timeout.flush(2000);
```
the test will fail without the patch. The second timeout.flush is crucial - without it, the watchFn spy doesn't get called, but *with* it, it gets called twice (without the patch).